deploy/packaging/redhat/redhat_build_rpms.py

```diff
@@ -219,6 +232,7 @@ def build():
     try:
         os.stat('/sign_keys/.gnupg')
```

Comment:
Same comment as above

Reply:
Same reply as above

Reply:
Use of `os.stat()` was the original code that existed prior to the recent RPM signing fixes.   Have simplified this as per suggestion above.